.github/workflows/migration-sql.yml

```diff
@@ -0,0 +1,84 @@
+name: Print alembic migrations SQL
+
+on:
+  pull_request:
+    branches: [master, sql-actions]
+
+jobs:
+  print_sql_upgrades:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.head.repo.full_name == github.repository
+
+    strategy:
+      matrix:
+        python-version: [3.8]
```

Comment:
Not needed; you can just specify `3.8` directly below when setting up Python

Reply:
Removed and added directly